### ISSUE TITLE
Uri source file at dataflow.spring.io for app import seems to be broken

### ISSUE THREAD

**Issue by kherpel** [spring-cloud/spring-cloud-dataflow#4849]
We are using the following dataflow-app-import container for SCDF app import:

```
  app-import:
    image: springcloud/openjdk:2.0.0.RELEASE
    container_name: dataflow-app-import
    networks:
      - inventory
    depends_on:
      - dataflow-server
    command: >
      /bin/sh -c "
        ./wait-for-it.sh -t 180 dataflow-server:9010;
        wget -qO- 'http://dataflow-server:9010/apps' --post-data='uri=${STREAM_APPS_URI:-https://dataflow.spring.io/kafka-maven-latest&force=true}';
        echo 'Stream apps imported'
        wget -qO- 'http://dataflow-server:9010/apps' --post-data='uri=${TASK_APPS_URI:-https://dataflow.spring.io/task-maven-latest&force=true}';
        echo 'Task apps imported'"
```

At the moment the import of apps fails with the following exception logged in the dataflow container log:

```
2022-03-04 22:38:27.760 ERROR 1 --- [nio-9010-exec-6] o.s.c.d.s.c.RestControllerAdvice         : Caught exception while handling a request

java.lang.IllegalArgumentException: Bad artifact coordinates org.springframework.cloud.stream.app:cdc-debezium-source-kafka:jar:metadata::3.2.0, expected format is <groupId>:<artifactId>[:<extension>[:<classifier>]]:<version>
```

I've checked the file at https://dataflow.spring.io/kafka-maven-latest and it seems there is an error in the second line with source.cdc-debezium.metadata:

```
source.cdc-debezium.metadata=maven://org.springframework.cloud.stream.app:cdc-debezium-source-kafka:jar:metadata::3.2.0
```

There are two colons before the version number but all other entries with extension/classifier and version are using just one (e.g., source.file.metadata in line 4).

Kind regards,
Kristian

**Comment by onobc** [comment]
@kherpel thanks for the report. You must have run into the issue shortly after I found it as well. :) 

It has been fixed in the stream-applications [2021.1.1](https://github.com/spring-cloud/stream-applications/releases/tag/v2021.1.1) release a few hours ago. 

Please verify that it is working for you now.

**Comment by kherpel** [comment]
@onobc Thanks for the information. The import works fine again.

Kind regards,
Kristian
